main.tf

```diff
@@ -41,22 +45,34 @@ resource "azurerm_role_definition" "subscription" {
 # permissions.
 resource "azurerm_role_definition" "resource_group" {
   for_each = toset(var.rsc_azure_features)
-  name        = "Rubrik Polaris RGRole ${each.key} - terraform - ${data.azurerm_subscription.current.subscription_id}"
-  description = "Rubrik Polaris Resource Group role for ${each.key} - Terraform Generated"
+
+  name        = "Rubrik Security Cloud RGRole ${each.key} - terraform - ${data.azurerm_subscription.current.subscription_id}"
+  description = "Rubrik Security Cloud Resource Group role for ${each.key} - Terraform Generated"
```

Comment:
@joeharlan the name and description provided here match what RSC currently creates during the with and without OAuth workflows. Please confirm that you want to move away from this standard.

Reply:
@DamaniN Same comment in this area as made in the other case you called out.  The names should be flexible to customer standards, so changes here to remove Polaris are compatible and immaterial.